airbyte-webapp/src/pages/OnboardingPage/OnboardingPage.tsx

```diff
@@ -46,7 +46,7 @@ const Footer = styled.div`
   display: flex;
   align-items: center;
   justify-content: center;
-  padding: 40px 20px 0;
+  margin-top: 100px; // need to move it a little bit down because of <StartWithDestination />, since it renders as a separate card with "position: absolute"
```

Comment:
```suggestion
  margin-top: auto; // need to move it a little bit down because of <StartWithDestination />, since it renders as a separate card with "position: absolute"
```
I think this fixes the issue @dizel852

Reply:
Err, not sure...
https://www.loom.com/share/6e3a2505f1d14697b67d352952037dc6

Reply:
Oh, I see.
Okay well I think this is probably good enough. Would you make a ticket to fix that styling? We can backlog and get to it eventually.